pages/privacy-policy.js

```diff
@@ -57,7 +57,7 @@ const privacyPolicy = () => {
                         </div>
                     </div>
                     <p id="3" className={[style.modal__button__para, style.none].join(' ')}>
-                        We have used most secured and open sourced programs to host our web app currently. We are using heroku and node and postgreSQL for our database. Therefore any data breach over heroku may lead to  data leakage on our website. In any such case, Vovoca or DevBucket does not holds any guarantee. We are currently working day and night and implementing methods to make vovoca a more secure app for your music and email and password. 
+                    We have used the most secured and open-sourced programs to host our web app currently. We are using Heroku  node and PostgreSQL for our database. Therefore any data breach over Heroku may lead to data leakage on our website. In any such case, Vovoca or DevBucket does not hold any guarantee. We are currently working day and night and implementing methods to make Vovoca a more secure app for your music and email and password.  
                     </p>
```

Comment:
replace content with 

At Vovoca, we take the security of your data seriously and have implemented stringent measures to protect your information. Our web app is hosted using the highly secure and open-source programs, Vercel, Node, and PostgreSQL, ensuring a robust infrastructure for data storage. However, it is important to note that in the unlikely event of a data breach on the Vercel platform, there is a possibility of data leakage on our website.

While we strive to maintain the highest level of security, we cannot provide an absolute guarantee against data breaches. In the event of such an occurrence, neither Vovoca nor DevBucket can be held responsible. However, please be assured that we are working tirelessly, dedicating ourselves day and night to implementing additional methods and security measures to further enhance the protection of your music, email, and password within the Vovoca app.

We value your trust in us, and the privacy and security of your data remain our top priority. We will continue to make every effort to ensure the utmost security of your information and provide you with a safe and secure platform for your musical experience.

Thank you for choosing Vovoca, and we appreciate your understanding and support as we continually strive to improve our security practices.